rxandroid/src/main/java/rx/android/schedulers/HandlerScheduler.java

```diff
@@ -13,122 +13,29 @@
  */
 package rx.android.schedulers;
 
-import java.util.concurrent.TimeUnit;
-
-import rx.Scheduler;
-import rx.Subscription;
-import rx.android.plugins.RxAndroidPlugins;
-import rx.android.plugins.RxAndroidSchedulersHook;
-import rx.exceptions.OnErrorNotImplementedException;
-import rx.functions.Action0;
-import rx.plugins.RxJavaPlugins;
-import rx.subscriptions.Subscriptions;
 import android.os.Handler;
-import android.os.Message;
+import android.os.Looper;
+import rx.Scheduler;
 
-/** A {@link Scheduler} backed by a {@link Handler}. */
-public final class HandlerScheduler extends Scheduler {
-    /** Create a {@link Scheduler} which uses {@code handler} to execute actions. */
+/**
+ * A {@link Scheduler} backed by a {@link Handler}.
+ *
+ * @deprecated Use {@link AndroidSchedulers#from(Looper)}.
+ */
+@Deprecated
```

Comment:
If we're going to another major revision for the next release (2.x), why not just remove it entirely?